Sources/ArcGISToolkit/Components/BasemapGallery/BasemapGalleryCell.swift

```diff
@@ -59,6 +59,10 @@ struct BasemapGalleryCell: View {
                     .multilineTextAlignment(.center)
                     .foregroundColor(item.hasError ? .secondary : .primary)
             }
+#if os(visionOS)
+            .contentShape(.hoverEffect, .rect(cornerRadius: 12))
```

Comment:
This is how I calculated this corner radius:
<img width="820" alt="Screenshot 2024-09-19 at 2 51 13 PM" src="https://github.com/user-attachments/assets/fa4f5d0a-b944-47e0-9b86-060376f2b0cc">